### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by r0mainK**
Small update to the README file, I lost a couple hours when I was building a Debug version of MinHashCuda for apollo, added it to the company guide [here](https://github.com/src-d/company/pull/132) and was prompted by Eiso to also incorporate some of it in the public doc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,12 +33,19 @@ Building
...
-It requires cudart, curand 7.5 and OpenMP 4.0 capable compiler.
+
+It requires cudart, curand 7.5, OpenMP 4.0 capable compiler and cmake >= 3.2.
 If [numpy](http://www.numpy.org/) headers are not found,
 specify the includes path with defining `NUMPY_INCLUDES`.
 If you do not want to build the Python native module, add `-D DISABLE_PYTHON=y`.
 If CUDA is not automatically found, add `-D CUDA_TOOLKIT_ROOT_DIR=/usr/local/cuda-8.0`
-(change the path to the actual one).
+(change the path to the actual one). 
+If you are building from a Docker container you may encounter the error:
```
Comment (vmarkovtsev) [inline README.md]: **in** a Docker container

the following error

File: README.md [hunk 2]
```diff
@@ -33,12 +33,19 @@ Building
...
+It requires cudart, curand 7.5, OpenMP 4.0 capable compiler and cmake >= 3.2.
 If [numpy](http://www.numpy.org/) headers are not found,
 specify the includes path with defining `NUMPY_INCLUDES`.
 If you do not want to build the Python native module, add `-D DISABLE_PYTHON=y`.
 If CUDA is not automatically found, add `-D CUDA_TOOLKIT_ROOT_DIR=/usr/local/cuda-8.0`
-(change the path to the actual one).
+(change the path to the actual one). 
+If you are building from a Docker container you may encounter the error: 
+`Could NOT find CUDA (missing: CUDA_TOOLKIT_ROOT_DIR CUDA_INCLUDE_DIRS CUDA_CUDART_LIBRARY)`
+This means need to get the rest of the CUDA toolkit, which can be installed like in the
```
Comment (vmarkovtsev) [inline README.md]: This means you need to install the rest

File: README.md [hunk 3]
```diff
@@ -33,12 +33,19 @@ Building
...
+`nvidia/cuda:8.0-devrel` DockerFile. 
+If you still run into `Could NOT find CUDA (missing: CUDA_INCLUDE_DIRS)` then run:
+ ```cp /usr/local/cuda/targets/x86_64-linux/include/* /usr/local/cuda/include/```
```
Comment (vmarkovtsev) [inline README.md]: It is very hacky to copy headers. Use a symbolic link instead

**Review by vmarkovtsev — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -33,12 +33,19 @@ Building
...
 If [numpy](http://www.numpy.org/) headers are not found,
 specify the includes path with defining `NUMPY_INCLUDES`.
 If you do not want to build the Python native module, add `-D DISABLE_PYTHON=y`.
 If CUDA is not automatically found, add `-D CUDA_TOOLKIT_ROOT_DIR=/usr/local/cuda-8.0`
-(change the path to the actual one).
+(change the path to the actual one). 
+If you are building in a Docker container you may encounter the following error: 
+`Could NOT find CUDA (missing: CUDA_TOOLKIT_ROOT_DIR CUDA_INCLUDE_DIRS CUDA_CUDART_LIBRARY)`
+This means you need to install the rest of the CUDA toolkit, which can be installed like in the
+`nvidia/cuda:8.0-devrel` DockerFile.
```
Comment (vmarkovtsev) [inline README.md]: It is `Dockerfile`

Comment (vmarkovtsev) [reply README.md]: Please link it

File: README.md [hunk 5]
```diff
@@ -33,12 +33,19 @@ Building
...
 If you do not want to build the Python native module, add `-D DISABLE_PYTHON=y`.
 If CUDA is not automatically found, add `-D CUDA_TOOLKIT_ROOT_DIR=/usr/local/cuda-8.0`
-(change the path to the actual one).
+(change the path to the actual one). 
+If you are building in a Docker container you may encounter the following error: 
+`Could NOT find CUDA (missing: CUDA_TOOLKIT_ROOT_DIR CUDA_INCLUDE_DIRS CUDA_CUDART_LIBRARY)`
+This means you need to install the rest of the CUDA toolkit, which can be installed like in the
+`nvidia/cuda:8.0-devrel` DockerFile. 
+If you still run into `Could NOT find CUDA (missing: CUDA_INCLUDE_DIRS)` then run:
+ ```ln  -s /usr/local/cuda/targets/x86_64-linux/include/* /usr/local/cuda/include/```
```
Comment (vmarkovtsev) [inline README.md]: Two whitespaces after `ln`, should be one

**Review by vmarkovtsev — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (r0mainK) [reply README.md]: done

File: README.md (hunk 5 above)
Comment (r0mainK) [reply README.md]: done

File: README.md [hunk 6]
```diff
@@ -33,12 +33,20 @@ Building
 ```
 cmake -DCMAKE_BUILD_TYPE=Release . && make
 ```
-It requires cudart, curand >=8.0 and OpenMP 4.0 compatible compiler (**that is, not gcc <=4.8**).
+
+It requires cudart, curand 7.5, OpenMP 4.0 compatible compiler (**that is, not gcc <=4.8**) and
```
Comment (vmarkovtsev) [inline README.md]: mismerge here: 7.5 vs >=8.0

**Review by vmarkovtsev — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (r0mainK) [reply README.md]: ah yeah that one went over my head, changing

**Review by vmarkovtsev — APPROVED** [review]

**Pull request merged**